packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -422,6 +423,17 @@ export interface IRootSummaryTreeWithStats extends ISummaryTreeWithStats {
 	gcStats?: IGCStats;
 }
 
+export interface IContainerRuntimeParams {
```

Comment:
IContainerRuntimeCreateParams is a more apt name. Also, please add a comment that indicates what it is.